docker-compose.override.yaml

```diff
@@ -5,6 +5,8 @@ version: '3.7'
 
 services:
   ffc-demo-web:
+    build:
+      target: development
```

Comment:
The nodemon watch isn't firing on file changes on my system. Everything appears fine, the folder is mounted OK. Could someone verify it's OK on their system, to see if it's a WSL related issue?

Reply:
Working OK now, not sure what the issue was